src/corelibs/U2Gui/src/OpenViewTask.cpp

```diff
@@ -84,13 +84,16 @@ static Task* createOpenViewTask(const MultiGSelection& ms) {
             ls.removeAll(f);
         }
     }
-
+    Task* t = nullptr;
+    if (QApplication::activeModalWidget() != nullptr) {
+        coreLog.info(QObject::tr("Unable to open view because of active modal widget."));
+        return t;
+    }
     if (ls.size() == 1) {
```

Comment:
nit: all lines below can be a 1 liner: return ls.size() == 1 ?  ls[0]->createViewTask()  : nullptr;